mcgill-augmented-reality-frontend/api.js

```diff
@@ -0,0 +1,66 @@
+import * as crypto from 'crypto-js';
+import Base64 from 'crypto-js/enc-base64';
+
+const {BACKEND, USERNAME, PASSWORD} = process.env;
```

Comment:
does this line work running expo on an actual phone and emulator? haven't been able to get it to work on my end